src/auth/AuthContainer.test.tsx

```diff
@@ -23,6 +23,20 @@ jest.mock('react-notifications-component', () => {
   };
 });
 
+jest.mock('src/libs/state', () => {
+  const state = jest.requireActual('src/libs/state');
+  return {
+    ...state,
+    oidcStore: {
+      ...state.oidcStore,
+      get: jest.fn().mockReturnValue({
+        ...state.oidcStore.get,
+        userManager: { getUser: jest.fn() },
+      }),
+    },
```

Comment:
In cases like this, where one method of an export is being mocked, I suspect we could reduce the boilerplate required by removing the module mock, importing the `oidcStore` and then using `jest.spyOn` to replace the desired method.